### DISCUSSION TITLE
Iframe and merge cells

### DISCUSSION THREAD

**Discussion by tpbillund** [AppFlowy-IO/AppFlowy#3506] [General]
Nice app/software.

Question:
- Is it/can it be, possible to merge cells/rows in a table?
- Is it/can it be, possible to add a iframe, with at link/online document in it?
- Is it/can it be, possible to add a local pdf/doc file as "Print-out" so you can read the pdf, from a site?

**Comment by annieappflowy** [comment]
Hi @tpbillund , thanks for your questions. We don't support the features you mentioned above. However, we do plan to support linking an online doc and exporting as a PDF format.
